src/System.Private.ServiceModel/src/System/IdentityModel/Tokens/SamlSecurityToken.cs

```diff
@@ -0,0 +1,124 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Text;
+using System.Xml.Serialization;
+using System.Xml;
+using System.Xml.Schema;
+using System.CodeDom;
+using System.Runtime.Serialization;
+using System.Globalization;
+using System.Threading;
+using System.IdentityModel.Selectors;
+using System.IdentityModel.Policy;
+using System.Reflection;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.IO;
+
+namespace System.IdentityModel.Tokens
+{
+    public class SamlSecurityToken : SecurityToken
+    {
+        SamlAssertion assertion;
+
+        protected SamlSecurityToken()
+        {
+        }
+
+        public SamlSecurityToken(SamlAssertion assertion)
+        {
+            Initialize(assertion);
+        }
+
+        protected void Initialize(SamlAssertion assertion)
+        {
+            if (assertion == null)
+                throw ServiceModel.DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("assertion");
```

Comment:
>ServiceModel [](start = 22, length = 12)

Consider `using DiagnosticUtility = System.ServiceModel.DiagnosticUtility ... `. Will not further comment on these